build/Dockerfile

```diff
@@ -9,8 +9,9 @@ ARG OC_CHANNEL=stable
 
 
 RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
-    dnf install -y jq git python3 unzip chromium chromedriver redhat-lsb-core &&\
-    dnf clean all
+    dnf install -y jq git unzip chromium chromedriver redhat-lsb-core &&\
+    dnf clean all && \
+    yum install -y python38
```

Comment:
why not do all installs in one line?

Reply:
I find this way more readable. Would there be any benefits in doing in one line?

Reply:
eh, not really, other than the fact that I would put the `clean all` at the end of all installs.
I would also either always use `dnf` or always use `yum`

Reply:
Actually, do you even need the clean all here?

Reply:
so, I replaced yum with dnf. 

About the `clean all`, it helps in reducing the size of the container (even though this docker file will be still changed to optimize the final size of the container)